src/execution_plan/ops/op_node_by_id_seek.c

```diff
@@ -130,6 +136,18 @@ static OpResult NodeByIdSeekReset(OpBase *ctx) {
 	return OP_OK;
 }
 
+static OpBase *NodeByIdSeekClone(const ExecutionPlan *plan, const OpBase *opBase) {
+	assert(opBase->type == OPType_NODE_BY_ID_SEEK);
+	NodeByIdSeek *op = (NodeByIdSeek *)op;
+	UnsignedRange range;
+	range.min = op->minId;
+	range.max = op->maxId;
+	// The range in the original op is set during construction, the clone range is inclusive.
+	range.include_min = true;
+	range.include_max = true;
```

Comment:
I don't understand why do we set `include_min` and `include_max` to true.

Reply:
the range is decided in the origin op, so we need to include its min and max

Reply:
Can you point me to where does the range is decided?